src/ui/components/shared/OnboardingModal/SingleInviteModal.tsx

```diff
@@ -4,8 +4,10 @@ import { connect, ConnectedProps } from "react-redux";
 import * as actions from "ui/actions/app";
 import hooks from "ui/hooks";
 import { PendingWorkspaceInvitation } from "ui/types";
+
```

Comment:
Might be worth us looking into a pre-commit hook that runs prettier to avoid noisy PRs. I personally like to have format on save running in VS Code.